hack/util.sh

```diff
@@ -382,10 +382,12 @@ function kill_all_processes()
   fi
 
   pids=($(jobs -pr))
-  for i in ${pids[@]}; do
-    ps --ppid=${i} | xargs $sudo kill &> /dev/null
-    $sudo kill ${i} &> /dev/null &> /dev/null
-  done
+  if [[ -n "${pids-}" ]]; then
```

Comment:
My mistake, I thought it was unnecessary (I thought I even tested it and it worked—weird).  You could simply use `${pids[@]-}` below though.